aws-api-appsync/src/main/java/com/amplifyframework/core/model/query/predicate/GsonPredicateAdapters.java

```diff
@@ -33,13 +34,15 @@ public final class GsonPredicateAdapters {
     private GsonPredicateAdapters() {}
```

Comment:
Can you explain why the change is needed in this class? Its not obvious from the review why we are having to change serialization code around QueryOperator and QueryPredicate as neither of these files changed.